### PR TITLE
Update devtools-usage.md

### PR DESCRIPTION
**Pull request by NickCoolii**
Changed content of config from config.ini to config.php

### CODE REVIEW

File: en/devtools-usage.md [hunk 1]
```diff
@@ -162,19 +162,41 @@ When a project is generated using developer tools. A configuration file can be f
...
-[phalcon]
-controllersDir = "../app/controllers/"
-modelsDir      = "../app/models/"
-viewsDir       = "../app/views/"
-baseUri        = "/store/"
+```php
+<?php
+/*
+ * Modified: prepend directory path of current file, because of this file own different ENV under between Apache and command line.
+ * NOTE: please remove this comment.
```
Comment (ninjapanzer) [inline en/devtools-usage.md]: I am not sure I understand the meaning of these comments. Can you give me a little more detail.

**Review by ninjapanzer — COMMENTED** [review]
There is a comment here that is a little confusing. Lets try to clean it up.

File: en/devtools-usage.md (hunk 1 above)
Comment (sergeyklay) [reply en/devtools-usage.md]: ping @NickCoolii

Comment (NickCoolii) [reply en/devtools-usage.md]: @ninjapanzer @sergeyklay 
this comment was in generated by Phalcon config.

**Comment by Ultimater** [comment]
Just noticed this PR existed and attempts to solve the same problem pointed out in my PR:
https://github.com/phalcon/docs/pull/1649

**Comment by Ultimater** [comment]
Note this PR attempt to swap from config.ini to config.php and has the following issue:

issues

File: en/devtools-usage.md (hunk 1 above)
Comment (sergeyklay) [reply en/devtools-usage.md]: @NickCoolii Just remove this comment. We'll try to sort out a bit later. Sorry it took this long

**Comment by sergeyklay** [comment]
@NickCoolii This branch has conflicts that must be resolved. Could you please rebase

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**